addons/website/static/src/js/content/auto_hide_menu.js

```diff
@@ -7,33 +7,27 @@ import { initAutoMoreMenu } from '@web/legacy/js/core/menu';
  */
 document.addEventListener('DOMContentLoaded', async () => {
     const header = document.querySelector('header#top');
-    if (!header || header.classList.contains('o_no_autohide_menu')) {
-        return;
-    }
-    const topMenu = header.querySelector('#top_menu');
-    const unfoldable = '.divider, .divider ~ li, .o_no_autohide_item';
-    const excludedImagesSelector = '.o_mega_menu, .o_offcanvas_logo_container, .dropdown-menu .o_lang_flag';
-    const excludedImages = [...header.querySelectorAll(excludedImagesSelector)];
-    const images = [...header.querySelectorAll('img')].filter((img) => {
-        excludedImages.forEach(node => {
-            if (node.contains(img)) {
-                return false;
-            }
-        });
-        return img.matches && !img.matches(excludedImagesSelector);
-    });
-    const defs = images.map((image) => {
-        if (image.complete) {
-            return null;
+    if (header) {
```

Comment:
Do you need that if ?

Reply:
Actually I added it because I had some errors on runbot for some cases where `const header = document.querySelector('header#top');` returns `null` and then `const topMenu = header.querySelector('#top_menu');`
triggers a traceback.